### PR TITLE
CFD-433 Request a group : The theme for this page should be kapablo, not seven

### PR DESCRIPTION
**Pull request by HelenaEksler**
https://issuetracker.amplexor.com/jira/browse/CFD-433

### CODE REVIEW

**Comment by HelenaEksler** [comment]
selection_308
selection_309

**Comment by HelenaEksler** [comment]
@nader77 PR is ready.

File: capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module [hunk 1]
```diff
@@ -293,3 +293,19 @@ function c4m_content_group_submit_redirect(&$form, &$form_state) {
   // Redirect a user to homepage.
   $form_state['redirect'] = url('', array('absolute' => TRUE));
 }
+
+/**
+ * Implements hook_custom_theme().
+ *
+ * Set Kapablo theme for group creation page.
+ */
+function c4m_content_group_menu_alter(&$items) {
```
Comment (nader77) [inline capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module]: This is not custom_theme, Could we do it like:
```
/**
 * Implements hook_custom_theme
 */
function MYMODULE_custom_theme() {
  // Set path specific theme.
  if (current_path() == 'node/add/group') {
    return variable_get('custom_theme', 'theme_name');
  }
}
```

@amitaibu Which one is better ?

Comment (amitaibu) [inline capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module]: hook_custom_theme() seems more straightforward

Comment (zero2one) [inline capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module]: I don't believe this is the solution:

* The **settings for the Admin theme** already state that all content create/edit/delete forms **should be rendered in the Default theme (Kapablo)**.
* This **works for all node types** (including projects) **except Groups**.
* There is probably an overwrite already of this behaviour that breaks the proper theme for groups.
* The **group edit form does use the Kapablo theme**, it's only the create that has a problem.

File: capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module [hunk 2]
```diff
@@ -293,3 +293,19 @@ function c4m_content_group_submit_redirect(&$form, &$form_state) {
   // Redirect a user to homepage.
   $form_state['redirect'] = url('', array('absolute' => TRUE));
 }
+
+/**
+ * Implements hook_menu_alter().
+ *
+ * Set Kapablo theme for group creation page.
```
Comment (zero2one) [inline capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module]: I don't believe this is the solution:

The settings for the Admin theme already state that all content create/edit/delete forms should be rendered in the Default theme (Kapablo).
This works for all node types (including projects) except Groups.
There is probably an overwrite already of this behaviour that breaks the proper theme for groups.
The group edit form does use the Kapablo theme, it's only the create that has a problem.

**Pull request merged**